ocs_ci/framework/conf/default_config.yaml

```diff
@@ -45,7 +45,7 @@ DEPLOYMENT:
   # ocs_csv_channel: "alpha"
   # you can overwrite the image for ocs operator catalog source by following parameter:
   # ocs_registry_image: "quay.io/rhceph-dev/ocs-registry:4.2-32.9b6c93e.master"
-  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-registry:latest"
+  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-registry:4.2-rc1"
```

Comment:
would have been better with rc-latest label.

Reply:
Probably it would but we don't have such tag https://quay.io/repository/rhceph-dev/ocs-registry?tag=latest&tab=tags  I can talk to Boris about this tag, but let's merge this one to make sure we installing RC as default and then we can change to some different tag once we will have.